src/components/menu/Aidrdrop/AirdropModal.tsx

```diff
@@ -0,0 +1,300 @@
+          backdrop-filter: blur(10px);
+          padding: 0 14px !important;
+        }
+      `}</style>
+    </div>
+  );
+}
+function _ContentElement(_props: React.ComponentPropsWithRef<'div'>, children: React.ReactNode) {
+  const { className, style, ...props } = _props;
+  return (
+    <div className={clsx(className, 'AirdropModal__Content')} {...props}>
+      {children}
+      <style jsx={true} global={true}>{`
+        .AirdropModal__Content {
+          width: 560px;
+          max-width: 100% !important;
+          padding: 30px !important;
+          background: #181a20;
+          border-radius: 12px;
+          color: white;
+          align-items: flex-start;
+          flex-direction: column;
+        }
+      `}</style>
+    </div>
+  );
+}
+
+function SpaceLine({ style }: { style?: CSSProperties }) {
+  return (
+    <>
+      <hr className="AirdropModal__space-line" style={style} />
+      <style jsx={true} global={true}>{`
+        .AirdropModal {
+          &__space-line {
+            width: 100%;
+            height: 1px;
+            border: 0;
+            border-radius: 9999px;
+            background: rgba(255, 255, 255, 0.2);
+          }
+        }
+      `}</style>
+    </>
+  );
+}
+
+function AirdropModal({ onRequestClose }: { onRequestClose: () => void }) {
+  const intl = useIntl();
+
+  const { currentAccount } = useUserWalletDataContext();
+  const [contract, setContract] = useState<ethers.Contract | null>(null);
+  const [claimableAmountRaw, setClaimableAmountRaw] = useState('0');
+  const [data, setData] = useState<{
+    totalAmount: string;
+    pendingAmount: string;
+    claimableAmount: string;
+  }>({
+    totalAmount: '-',
+    pendingAmount: '-',
+    claimableAmount: '-',
+  });
+  useEffect(() => {
+    const provider = new ethers.providers.Web3Provider((window as any).ethereum);
+    const signer = provider.getSigner();
+    const _contract = new ethers.Contract(CORN_AIRDROP_ADDRESS, AirdropAbi, signer);
+    setContract(_contract);
+    Promise.all([
+      _contract.getTotalAmount(currentAccount),
+      _contract.getPendingAmount(currentAccount),
+      _contract.getClaimableAmount(currentAccount),
+    ]).then(([_totalAmount, _pendingAmount, _claimableAmount]) => {
+      setClaimableAmountRaw(_claimableAmount.toString());
+      const totalAmount = valueToBigNumber(
+        normalize(valueToBigNumber(_totalAmount.toString()).toString(), CORN_DECIMALS)
+      ).toFixed(4, BigNumber.ROUND_DOWN);
+      const pendingAmount = valueToBigNumber(
+        normalize(valueToBigNumber(_pendingAmount.toString()).toString(), CORN_DECIMALS)
+      ).toFixed(4, BigNumber.ROUND_DOWN);
+      const claimableAmount = valueToBigNumber(
+        normalize(valueToBigNumber(_claimableAmount.toString()).toString(), CORN_DECIMALS)
+      ).toFixed(4, BigNumber.ROUND_DOWN);
+      setData({
+        totalAmount,
+        pendingAmount,
+        claimableAmount,
+      });
+      console.log({
```

Comment:
We can remove the log if not needed any more.

Reply:
need to keep it before merge. Test engineers need to test with this data